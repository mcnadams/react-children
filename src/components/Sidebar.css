.sidebar {
  display: flex;
  flex-direction: column;

  background: black;
  color: fuschia;
  font: white;

  text-align: center;

  a {
    text-decoration: none;
    color: orchid;

  }
}