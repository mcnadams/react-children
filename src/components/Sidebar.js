import React from 'react';
import PropTypes from 'prop-types';
import styles from './Sidebar.css';

function Sidebar({ children }) {
  console.log(children);
  return (
    <nav className={styles.sidebar}> 
      {children}
    </nav>
  );
}

Sidebar.propTypes = {
  children: PropTypes.node
};

export default Sidebar;
