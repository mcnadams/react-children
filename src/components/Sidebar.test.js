import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('Sidebar', () => {

  it('renders a sidebar', () => {
    const wrapper = shallow(<Sidebar />);
    expect(wrapper).toMatchSnapshot();
  });

});
