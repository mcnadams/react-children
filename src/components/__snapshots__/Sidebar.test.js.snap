// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar renders a sidebar 1`] = `
<nav
  className="sidebar"
/>
`;
